Login.css

```diff
@@ -0,0 +1,91 @@
+body{background-color: white;
+    color : black;
+    font-family: "comic sans ms";
+    
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items :center;
+}
+.nav-container{
+    background-color: rgb(188, 251, 62);
+ align-items: center;
+justify-content: space-between;
+width : 97vw;
+
+}
+
+.nav-footer{
+    align-items: center;
+    width : 20%;
+}
+.btn{
+    margin : 0px;
+    background-color: lightgray;
+}
+
+.nav-main{display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left : 250%;
+    
+}
+
+.nav-input{text-align: center;
+    font-family: "comic sans ms";
+    color : black;
+    font-size: 1.25rem;
+    width : 500px;
```

Comment:
convert to rems here for responsiveness.....tip: never use px if the value is greater than 5 or 10px depending on use-case